apps/edxapp/vars/all/main.yml

```diff
@@ -7,7 +7,7 @@ edxapp_preview_host: "preview.{{ project_name}}.{{ domain_name }}"
 
 # -- edxapp (cms/lms)
 edxapp_image_name: "fundocker/edxapp"
-edxapp_image_tag: "hawthorn.1-oee-3.1.2"
+edxapp_image_tag: "hawthorn.1-oee-3.2.0"
```

Comment:
Actually i need to release a new edxapp image and pinning it here before merging this PR